src/source-af-packet.c

```diff
@@ -179,69 +179,87 @@ enum {
 
 union thdr {
     struct tpacket2_hdr *h2;
+    struct tpacket3_hdr *h3;
     void *raw;
 };
 
+struct block_desc {
```

Comment:
At least on Linux, linux/if_packet defines tpacket_block_desc, why are you defining a similar struct yourself? Is this for compatibility reasons?